packages/tocco-ui/src/Typography/Font.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import {injectGlobal} from 'styled-components'
+import {themeGet} from 'styled-system'
+
+const InjectFontRoboto = props => {
+  const fontFamilySansSerif = themeGet('fontFamily.sansSerif', 'false')(props)
+  const fontFamilyMonospace = themeGet('fontFamily.monospace', 'false')(props)
+
+  if (fontFamilySansSerif.includes('Roboto')) {
+    // eslint-disable-next-line chai-friendly/no-unused-expressions
```

Comment:
https://eslint.org/docs/rules/no-unused-expressions

take a look at allowTaggedTemplates. should fix this without eslint-disable

Reply:
Declaring "chai-friendly/no-unused-expressions": [2, { "allowTaggedTemplates": true }] in /.eslintrc would make disabling this specific rule dispensable. Whereas currently linting is disabled for only one line, we are about to allow Tagged Template Literals globally without assignment.

injectGlobal is used rarely. Tagged Template Literals are normally nested in functions or are assigned to variable, which does not trigger any linting rule. Hence I prefer the current implementation (being explicit).